src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -113,165 +110,173 @@ const MapView: React.FC<MapViewProps> = ({
     if (mapElement && treePopupElement) {
       const markersArray: google.maps.Marker[] = [];
 
-      LOADER.load().then(() => {
-        map = new google.maps.Map(mapElement, {
-          center: BOSTON,
-          zoom: 12,
-          fullscreenControl: false,
-          mapTypeControl: false,
-          // mapId: mapId,
-          restriction: {
-            latLngBounds: BOSTON_BOUNDS,
-            strictBounds: false,
-          },
-        });
+      LOADER.load()
+        .then(() => {
+          map = new google.maps.Map(mapElement, {
```

Comment:
Theres _SO_ much code in this anonymous function 🤯 . We really need to pull most of this out. I could barely even find the .catch

Reply:
I could barely find the place to put the catch 😂 . The next ticket for the map is pulling everything out and testing